src/palace/manager/sqlalchemy/model/work.py

```diff
@@ -1649,62 +1658,68 @@ def _set_value(parent, key, target):
 
         result["contributors"] = []
         if doc.presentation_edition and doc.presentation_edition.contributions:
-            for item in doc.presentation_edition.contributions:
+            for contribution in doc.presentation_edition.contributions:
```

Comment:
All these for loop variable were updated because now that we have proper types reusing `item` in each loop gives us type errors. So I switched out the loop variable for each one.